api/routes/news.js

```diff
@@ -0,0 +1,90 @@
+const Joi = require('@hapi/joi');
+const express = require('express');
+const router = express.Router();
+const newsArray = [
+    {
+        id: 1,
+        name: "Nitin"
+    },
+    {
+        id: 2,
+        name: "Slava"
+    },
+    {
+        id: 3,
+        name: "Jack"
+    },
+    {
+        id: 4,
+        name: "Kohli"
+    }
+];
+
+const errorGenerator = (text, status)=> {
```

Comment:
class CustomError extends Error {
  constructor(text, status){
    super(text)
    
    this.status = status
  }
  
}


Then just do throw new CustomError(text, status)

Reply:
Resolved